models/fastText_expirements.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>This would be a good spot to do some EDA on the labels. Like you could output what are the top N most labels. What there counts are, How many issues are missing labels, etc. </p><p> </p><p>also how did you land on n=30? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
yea I can add a bunch of EDA, and I'm probably better off setting a # of samples cutoff instead of the top 30